lebedigital/calibration/calibrationWorkflow.py

```diff
@@ -0,0 +1,202 @@
+    # skip_last = 145
+    # skip_init = 330
+    #experiment_name = 'Wolf 8.2 Probe 1'
+    # exp_output = load_experimental_data(experiment_name, skip_init, skip_last, path=path_csv)
+    # -- if KG is present, comment the above
+    #path_to_KG = '../usecases/MinimumWorkingExample/emodul/knowledge_graphs/'
+    if KnowledgeGraph:
+        path = path_to_KG + '/' + experiment_name + '.ttl'
+        exp_output = query_KG(path=path)
+        plt.plot(exp_output['stress'], exp_output['displacement'] / exp_output['height'])  # checking loading data
+    else:
+        path_csv = path_to_KG # if KG not present, path takes path of CSV
+        #path_csv = os.path.dirname(Path(__file__)) + '/Wolf 8.2 Probe 1.csv'
+        skip_last = 145
+        skip_init = 330
+        exp_output = load_experimental_data(experiment_name, skip_init, skip_last, path=path_csv)
+
+    # This generated Latex errors!!! Maybe we require some extra package
+    #plt.show()
+    # ========================================
+    #       Set Numerical values
+    # ========================================
+    # "uninformed" Normal prior for the E modulus
+    loc_E = 100  # kN/mm2
+    scale_E = 100  # this can be inferred too
+
+    # Uniform prior for experimental noise/model discrepancy
+    low_sigma = 0
+    high_sigma = 0.005
+
+    # =========================================
+    #       Define the Inference Problem
+    # =========================================
+
+    # initialize the inverse problem with a useful name
+    problem = InverseProblem("compression test calibration")
+
+    # add all parameters to the problem
+    problem.add_parameter('E', 'model',
+                          tex="$E$",
+                          info="Slope of the graph",
+                          prior=('normal', {'mean': loc_E,
+                                            'std': scale_E}))
+
+    problem.add_parameter('sigma', 'likelihood',
+                          tex=r"$\sigma$",
+                          info="Std. dev, of 0-mean noise model",
+                          prior=('uniform', {'low': low_sigma,
+                                             'high': high_sigma}))
+
+    # Load the forward model and add it to the Inverse problem
+    linear_elasticity = LinearElasticityCylinder("linear_elasticity_cylinder")
+    problem.add_forward_model(linear_elasticity)
+
+    # ============================================
+    #     Add test data to the Inverse Problem
+    # ============================================
+
+    y_test = exp_output['stress'] * (np.pi * (exp_output['diameter'] / 2) ** 2)  # in kN
+    # add the experimental data
+    problem.add_experiment(
+        experiment_name,
+        fwd_model_name="linear_elasticity_cylinder",
+        sensor_values={
+            'nu': 0.2,
+            'height': exp_output['height'],
+            'radius': exp_output['diameter'] / 2,
+            'displacement_list': exp_output['displacement'],
```

Comment:
same here, what is that displacement (the actual deformation of the specimen (30cm), or the relative displacement of the central part (10cm)

Reply:
see above, the data is only for the inner 10cm, the total height is 30cm, but the gauge length is 10cm.

Reply:
The simulation has been changed to fixed 10cm "height" and the boundary conditions changed to match the unconfined conditions.